.gitignore

```diff
@@ -3,6 +3,9 @@ logs
 *.log
 npm-debug.log*
 
+# Lock files
+package-lock.json
```

Comment:
I think it makes sense for us to include these, to allow users to "guarantee" they are using the right dependencies.

Reply:
I thought the same but wasn't certain that's why added to the ignore list.